README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
+  name                        = "admin"
+  namespace                   = "${var.namespace}"
+  stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
```

Comment:
Default value `associate_public_ip_address=true` is defined in [tf_instance](https://github.com/cloudposse/tf_instance).
Generally, `tf_instance` shouldn't have public IP, because it can be located only within Private Subnet.